include/Types.hpp

```diff
@@ -27,4 +28,5 @@ namespace blitzdg {
     using index_tensor3_type = tensor3_type<index_type>;
     using real_tensor3_type = tensor3_type<real_type>;
 
+    using index_hashmap = std::unordered_map<blitzdg::index_type, blitzdg::index_vector_type>;
 }
```

Comment:
Is this a type you intend on using elsewhere? It's not really a fundamental scalar, vector, or matrix type, so I am not sure that it belongs here.

Reply:
I went back and forth on this one. It's going to be needed in the solvers themselves to pull in the boundary conditions hashmap, so I thought putting it in the blitzdg namespace made sense.